usaspending_api/awards/tests/test_awards.py

```diff
@@ -29,7 +29,7 @@ def test_award_endpoint(client):
             data=json.dumps(
                 {
                     "filters": [
-                        {"field": "funding_agency__toptier_agency__fpds_code", "operation": "equals", "value": "0300"}
+                        {"field": "funding_agency__toptier_agency__cgac_code", "operation": "equals", "value": "0300"}
```

Comment:
is cgac code actually equivalent of whatever fpds code used to be? I doubt that this test cared was a fpds_code was, but still worth thinking about.